docs/README.md

```diff
@@ -28,33 +28,31 @@ The MeasurementLink Support for LabVIEW packages enable measurement developers t
 
 Set up the following software before installing the given packages.
 
-- VI Package Manager 2020
+- InstrumentStudio 2023 Q1 or newer
+- MeasurementLink 2023 Q1 or newer
 - LabVIEW 2020 SP1 or newer
-- InstrumentStudio - 2023 Q1 or newer
-- MeasurementLink - 2023 Q1 or newer
+  - VI Package Manager 2020 or newer
 
 ---
 
 ## Software support
```

Comment:
This section just appears like a repeat of the previous. I suggest we combine them. Not sure what the header should be.

Reply:
Agree.  Maybe we should just remove the "Dependencies" section.  We already cover version compatibility in "Software support", and we follow that with "Installation", which lists everything.

Reply:
Update coming.

Reply:
... and while I'm here, it looks like there are a few stale quick-links in the header.  I'll clean those up too.